pkg/backend/nvme_namespace.go

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I think we should postpone `NVMfRemoteNamespace` to a later PR
let's focus on PATH in this work ?